src/compiler/crystal/tools/init.cr

```diff
@@ -64,12 +65,29 @@ module Crystal
       fetch_required_parameter(opts, args, "NAME")
     end
 
+    def self.directory_valid?(directory)
+      if File.file?(directory)
+        puts "file #{directory} already exists"
```

Comment:
The error message should not only contain the failure cause, but also what this means in context. Like: `cannot initialize Crystal project: #{directory} it is a file`